core/src/accounts_hash_verifier.rs

```diff
@@ -109,7 +115,28 @@ impl AccountsHashVerifier {
                         ("enqueued-time-us", enqueued_time.as_micros(), i64),
                         ("handling-time-us", handling_time_us, i64),
                     );
+
+                    // Done processing the current snapshot, so the current snapshot dir
+                    // has been converted to POST state.  It is the time to update
+                    // last_snapshot_storages to release the reference counts for the
+                    // previous POST snapshot dir, and save the new ones for the new
+                    // POST snapshot dir.
+                    last_snapshot_storages = Some(snapshot_storages);
+                    debug!(
```

Comment:
I see they were just moved, but I'm curious about why this debug message is different from the one on line 133.

I think if we want to make the logging different so we can determine which is from a **new** snapshot, then we should include that in the log message.

(feel free to do this in a separate PR - it's not related to the move of this code)

Reply:
I copied them from the same positions in ABS, one in the loop, one out of the loop.  The in-loop and out-loop messages may not be the same.

Reply:
Ok, these two messages mean the same thing.  I will make them the same.

Reply:
updated it.